lib/libwa/src/my_wordarray_len.c

```diff
@@ -7,6 +7,18 @@
 
 #include <stddef.h>
 
+int my_wordarray_size(char **wordarray)
+{
+    int i = 0;
+    int y = 0;
+
+    if (wordarray == NULL)
+        return (0);
+    for (; wordarray[i] != NULL; i++)
+        for(; wordarray[i][y] != '\0'; y++);
```

Comment:
this can crash if there are two elements in the word array
because if the first str len = 10
and the second = 5
y = 10
and access of the second element[10] will seg

Reply:
it's ok i changed